src/views/monitoring/alert/modules/alert-detail/AlertDetailHeader.vue

```diff
@@ -83,30 +95,59 @@ export default {
     props: {
         id: {
             type: String,
-            default: null,
+            default: undefined,
         },
         alertData: {
             type: Object,
-            default: null,
+            default: () => ({}),
         },
     },
-    setup(props: PropsType) {
+    setup(props: PropsType, { emit }) {
         const state: UnwrapRef<HeaderState> = reactive({
-            alertState: props.alertData.state,
-            alertUrgency: props.alertData.urgency,
+            alertState: props.alertData?.state,
+            alertUrgency: props.alertData?.urgency,
             reassignModalVisible: false,
-            duration: calculateTime(props.alertData.created_at),
+            duration: calculateTime(props.alertData?.created_at),
         });
 
         const onClickReassign = () => {
             state.reassignModalVisible = true;
         };
 
+        const onConfirmReassign = () => {
+            emit('confirm');
+        };
+
+        const changeAlertState = async () => {
+            try {
+                await SpaceConnector.client.monitoring.alert.updateState({
+                    alert_id: props.id,
+                    state: state.alertState,
+                });
+            } catch (e) {
+                console.error(e);
+            }
+        };
+
+        const changeAlertUrgency = async () => {
+            try {
+                await SpaceConnector.client.monitoring.alert.update({
+                    alert_id: props.id,
+                    urgency: state.alertUrgency,
+                });
+            } catch (e) {
+                console.error(e);
+            }
+        };
```

Comment:
옷 요 칭구들은 success/fail 토스트 안띄워도 되나요??

Reply:
`v-model` 로 `alertState`, `alertUrgency` 가 바인딩 되어있는데, 변경된 alertState 가 반영된 상태에서 해당 함수들이 불려진다는 것을 장담키는 어려울 것 같아요.
사실 이 부분은 저도 확인한 바는 없는데, 관련 이슈로 애먹은 적이 많아서 혹시나 하는 생각이 듭니다...ㅠ

```suggestion
<p-select-dropdown :value="alertState" ... >
<p-select-dropdown :value="alertUrgency" ... >

...

        const changeAlertState = async (alertState: 타입정의) => {
            state.alertState = alertState;
            try {
                await SpaceConnector.client.monitoring.alert.updateState({
                    alert_id: props.id,
                    state: alertState,
                });
            } catch (e) {
                console.error(e);
            }
        };

        const changeAlertUrgency = async (alertUrgency: 타입정의) => {
            state.alertUrgency = alertUrgency;
            try {
                await SpaceConnector.client.monitoring.alert.update({
                    alert_id: props.id,
                    urgency: state.alertUrgency,
                });
            } catch (e) {
                console.error(e);
            }
        };
```

이렇게 하는 것이 좀 더 안전한 방법 같습니다...만! 
만약 보장된다라고 한다면, 기존 방식도 아주 좋습니당

Reply:
아까 저 value로 했더니 뭔가 받아오지 못했던거같아서??ㅠㅠ 이렇게 했는데, 반영해보고 작동하면 수정하겠습니다!!

Reply:
>> 옷 요 칭구들은 success/fail 토스트 안띄워도 되나요??

앜! 적용해놓을게요 ㅎㅎ